manifests/u/UnknownSKL/Greenlight/2.1.1/UnknownSKL.Greenlight.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: UnknownSKL.Greenlight
+PackageVersion: 2.1.1
+Installers:
+- InstallerUrl: https://github.com/unknownskl/greenlight/releases/download/v2.1.1/Greenlight-Setup-2.1.1.exe
+  Architecture: x86

```

```suggestion
  Architecture: x64
```
Shouldn't this be x64 architecture?